pages.de/linux/apt-file.md

```diff
@@ -1,6 +1,6 @@
 # apt-file
 
-> Suche nach Dateien in apt Paketen, auch in den nicht installierten.
+> Suche nach Datein in apt-Paketen, auch in den nicht-installierten.
```

Comment:
I'm pretty sure it's `Dateien`, not `Datein`.
```suggestion
> Suche nach Dateien in apt-Paketen, auch in den nicht-installierten.
```

Reply:
Oh well, that wasn't on purpose.